mm2src/common/executor/abortable_system/abortable_queue.rs

```diff
@@ -266,12 +320,35 @@ mod tests {
         };
         spawner.spawn_with_settings(fut2, settings.clone());
 
-        abortable_system.abort_all();
+        abortable_system.abort_all().unwrap();
 
         block_on(Timer::sleep(1.2));
         // `fut1` must not complete.
         assert!(unsafe { !F1_FINISHED });
         // `fut` must complete.
         assert!(unsafe { F2_FINISHED });
     }
+
+    #[test]
+    fn test_spawn_after_abort() {
+        static mut F1_FINISHED: bool = false;
```

Comment:
Why static needed here?

Reply:
It's just more convenient to use a static mut variable in tests than `Arc<AtomicBool>`.
Isn't it?